src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/InfluxDBHelper.java

```diff
@@ -341,13 +328,12 @@ public void ingestToInfluxDb(
             // Enable or disable using
             backupService.writeToBackup(payload, new URL(baseUrl), databaseName, retPolicyName);
             influxDB.write(databaseName, retPolicyName, InfluxDB.ConsistencyLevel.ONE, TimeUnit.SECONDS, payload);
-        }
-        catch(InfluxDBException.PointsBeyondRetentionPolicyException ex) {
```

Comment:
This original catch could be helpful actually. It indicates that a data point was too old to be valid within the requested retention policy. I'd like to see that caught still but a new `InvalidDataException` (for example) be thrown where the message should indicate "Timestamp too old".